CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing Guidelines
+1. Create a new branch for your contribution.
+2. Make sure your branch is up-to-date: `git pull upstream main`
+3. Make your changes.
+5. Push your changes to your fork.
+6. Squash commits before making a PR.
+7. Submit a pull request through GitHub.
```

Comment:
The issue: https://github.com/jackconryan/DartsElo/issues/5#issue-1915982105 is not referencing CONTRIBUTION.md or README.md documentation. it is referencing out of date documentation in the code. 

I'm closing this PR as it seems there is a lack of understanding in the code (see 'npm install' instruction in python project) and lack of understanding of the issue